resources/definitions/mingda_base.def.json

```
@@ -0,0 +1,265 @@
+
+        "machine_max_feedrate_x": { "value": 500 },
+        "machine_max_feedrate_y": { "value": 500 },
+        "machine_max_feedrate_z": { "value": 10 },
+        "machine_max_feedrate_e": { "value": 50 },
+
+        "machine_max_acceleration_x": { "value": 500 },
+        "machine_max_acceleration_y": { "value": 500 },
+        "machine_max_acceleration_z": { "value": 100 },
+        "machine_max_acceleration_e": { "value": 5000 },
+        "machine_acceleration": { "value": 500 },
+
+        "machine_max_jerk_xy": { "value": 10 },
+        "machine_max_jerk_z": { "value": 0.4 },
+        "machine_max_jerk_e": { "value": 5 },
+
+        "machine_heated_bed": { "default_value": true },
+
+        "material_diameter": { "default_value": 1.75 },
+
+        "acceleration_print": { "value": 500 },
+        "acceleration_travel": { "value": 500 },
+        "acceleration_travel_layer_0": { "value": "acceleration_travel" },
+        "acceleration_roofing": { "enabled": "acceleration_enabled and roofing_layer_count > 0 and top_layers > 0" },
+
+        "jerk_print": { "value": 8 },
+        "jerk_travel": { "value": "jerk_print" },
+        "jerk_travel_layer_0": { "value": "jerk_travel" },
+
+        "acceleration_enabled": { "value": false },
+        "jerk_enabled": { "value": false },
+
+        "speed_print": { "value": 60.0 } ,
+        "speed_infill": { "value": "speed_print" },
+        "speed_wall": { "value": "speed_print / 2" },
+        "speed_wall_0": { "value": "speed_wall" },
+        "speed_wall_x": { "value": "speed_wall" },
+        "speed_topbottom": { "value": "speed_print / 2" },
+        "speed_roofing": { "value": "speed_topbottom" },
+        "speed_travel": { "value": "150.0 if speed_print < 60 else 250.0 if speed_print > 100 else speed_print * 2.5" },
+        "speed_layer_0": { "value": 20.0 },
+        "speed_print_layer_0": { "value": "speed_layer_0" },
+        "speed_travel_layer_0": { "value": "100 if speed_layer_0 < 20 else 150 if speed_layer_0 > 30 else speed_layer_0 * 5" },
+        "speed_prime_tower": { "value": "speed_topbottom" },
+        "speed_support": { "value": "speed_wall_0" },
+        "speed_support_interface": { "value": "speed_topbottom" },
+        "speed_z_hop": { "value": 5 },
+
+        "skirt_brim_speed": { "value": "speed_layer_0" },
+
+        "line_width": { "value": "machine_nozzle_size * 1.1" },
+
+        "optimize_wall_printing_order": { "value": "True" },
+
+        "material_initial_print_temperature": { "value": "material_print_temperature" },
+        "material_final_print_temperature": { "value": "material_print_temperature" },
+        "material_flow": { "value": 100 },
+        "travel_compensate_overlapping_walls_0_enabled": { "value": "False" },
```

```suggestion
        "travel_compensate_overlapping_walls_0_enabled": { "value": false },
```
`"False"` will work, but it's better to use `false` instead.